centrallix/htmlgen/htdrv_menu.c

```diff
@@ -336,26 +344,26 @@ htmenuRender(pHtSession s, pWgtrNode menu, int z)
 	htrAddEventHandlerFunction(s, "document", "MOUSEDOWN", "mn", "mn_mousedown");
 
 	/** Beginning of code for menu **/
-	htrAddBodyItem_va(s,"<div id=\"mn%POSmain\">", id);
-	if (s->Capabilities.Dom0NS)
-	    htrAddBodyItem_va(s,"<body %STR>",bgstr);
-	htrAddBodyItem_va(s,"<div id=\"mn%POScontent\"><table cellspacing=\"0\" cellpadding=\"0\" border=\"0\" %STR>\n", id, s->Capabilities.Dom0NS?"":"width=\"100%\" height=\"100%\"");
+	htrAddBodyItem_va(s,"<div id=\"mn%POSmain\" class=\"menuMain\">", id);
+	//if (s->Capabilities.Dom0NS)
+	 //   htrAddBodyItem_va(s,"<body %STR>",bgstr);
+	htrAddBodyItem_va(s,"<div id=\"mn%POScontent\" class=\"menuContent zeroLeftTop\"><table cellspacing=\"0\" cellpadding=\"0\" border=\"0\" %STR>\n", id, s->Capabilities.Dom0NS?"":"width=\"100%\" height=\"100%\"");
 
 	/** Only draw border if it is NS4 **/
-	if (s->Capabilities.Dom0NS)
-	    {
-	    htrAddBodyItem(s,"<tr><td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"1\" width=\"1\"></td>");
-	    if (w != -1)
-		htrAddBodyItem_va(s,"<td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"1\" width=\"%POS\"></td>", w-2);
-	    else
-		htrAddBodyItem(s,"<td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"1\" width=\"1\"></td>");
-	    htrAddBodyItem(s,"<td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"1\" width=\"1\"></td></tr>\n");
-	    if (h != -1)
-		htrAddBodyItem_va(s,"<tr><td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"%POS\" width=\"1\"></td><td>", h-2);
-	    else
-		htrAddBodyItem(s,"<tr><td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"1\" width=\"1\"></td><td>");
-	    }
-	else
+//	if (s->Capabilities.Dom0NS)
+//	    {
+//	    htrAddBodyItem(s,"<tr><td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"1\" width=\"1\"></td>");
+//	    if (w != -1)
+//		htrAddBodyItem_va(s,"<td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"1\" width=\"%POS\"></td>", w-2);
+//	    else
+//		htrAddBodyItem(s,"<td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"1\" width=\"1\"></td>");
+//	    htrAddBodyItem(s,"<td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"1\" width=\"1\"></td></tr>\n");
+//	    if (h != -1)
+//		htrAddBodyItem_va(s,"<tr><td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"%POS\" width=\"1\"></td><td>", h-2);
+//	    else
+//		htrAddBodyItem(s,"<tr><td background=\"/sys/images/white_1x1.png\"><img src=\"/sys/images/trans_1.gif\" height=\"1\" width=\"1\"></td><td>");
+//	    }
+//	else
```

Comment:
The Dom0NS branch of this IF statement can be deleted entirely (not just commented out).  We can always go back to prior git revisions to retrieve the code if we ever need it for some reason.